Tests/DependencyInjection/ZenstruckCacheExtensionTest.php

```diff
@@ -36,19 +36,20 @@ public function testWithHost()
         $this->assertTrue($this->container->has('zenstruck_cache.sitemap_provider'));
     }
 
-    /**
-     * @expectedException \Symfony\Component\Config\Definition\Exception\InvalidConfigurationException
-     */
-    public function testMissingHost()
+    public function testWithHosts()
```

Comment:
Can you add a test for just adding `host` to ensure BC? Also, can you keep the exception test if `hosts` and `host` isn't defined?